.github/install/deploywithsampleapp.yaml

```diff
@@ -0,0 +1,15 @@
+name: istio
+services:
+  istioinstallation:
+    type: $(#use https://raw.githubusercontent.com/Revolyssup/service-mesh-patterns/import/samples/istioinstall.yaml)
+    settings: 
+        controlplaneNS: istio-system
+        namespace: default
+        istioVersion: 1.12.0
+        profile: minimal
+  bookinfo:
+    type: $(#use https://raw.githubusercontent.com/gr455/service-mesh-patterns/bookinfo-remove-k8s/samples/bookInfoPattern.yaml)
```

Comment:
We might want to have these under `meshery/service-mesh-patterns`

Reply:
yes please use the patterns from service-mesh-patterns/service-mesh-patterns only